### PR TITLE
Clean up current implementation of the common trips

### PR DESCRIPTION
**Pull request by shankari**
It is useless to try and debug the code unless it is actually understandable.
Therefore, we remove the old, obsolete code to improve code clarity.
This reveals that the tests had not been fixed to work with the new code and so were testing currently unused code.
Ended up fixing the tests as well.

Check out commit messages for more details.

### CODE REVIEW

**Comment by shankari** [comment]
@sunil07t, you can review this now

File: emission/user_model_josh/utility_model.py [hunk 1]
```diff
@@ -127,7 +126,7 @@ def get_top_choice_places(self, start_place, end_place):
         return self.get_top_choices_lat_lng(start, end)
 
     def get_all_trips(self, start, end, curr_time=None):
-        c = googlemaps.client.Client(GOOGLE_MAPS_KEY)
+        c = gmaps.client.Client(GOOGLE_MAPS_KEY)
```
Comment (sunil07t) [inline emission/user_model_josh/utility_model.py]: Line 507 has googlemaps instead of gmaps, does that need to be gmaps?

**Comment by shankari** [comment]
@sunil07t, good catch! Fixed.
I guess Josh didn't write any test cases for those functions...

**Comment by shankari** [comment]
common_trips_working_again

File: emission/tests/analysisTests/TestSimilarity.py [hunk 2]
```diff
@@ -86,10 +94,10 @@ def testDeleteBins(self):
             self.assertTrue(b == sim.num)
 
     def testElbowDistance(self):
-        start = Coordinate(47,-122)
-        end = Coordinate(47,-123)
-        now = datetime.datetime.now()
-        t = Trip(None, None, None, None, now, now, start, end)
+        start = [-122,47]
+        end = [-133,47]
```
Comment (sunil07t) [inline emission/tests/analysisTests/TestSimilarity.py]: is the x coordinate a typo or did you change it compared to old test? (-123 instead of -133)

Comment (shankari) [inline emission/tests/analysisTests/TestSimilarity.py]: Typo. Will fix along with anything else that you catch.
Also, note that I didn't write the original tests - they were written by a SUPERB student from the last year. So you too can write tests :)

**Comment by sunil07t** [comment]
That is it! Everything else is clear :+1:

File: emission/tests/analysisTests/TestSimilarity.py (hunk 2 above)
Comment (sunil07t) [inline emission/tests/analysisTests/TestSimilarity.py]: I can try to write the test codes after Goals tab :)

**Pull request merged**